core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -108,16 +108,7 @@ public boolean isDisabled() {
                 Strings.isNullOrEmpty(getContext().getInstrumentationKey()) || configuration.isTrackingDisabled();
     }
 
-    /**
-     * Sends the specified state of a user session to Application Insights.
-     * @param sessionState {@link com.microsoft.applicationinsights.telemetry.SessionState}
-     *                     value indicating the state of a user session.
-     */
-    public void trackSessionState(SessionState sessionState) {
```

Comment:
Should we keep it "deprecated" for at least one version to give some time for switching over to a new API?
Also, since this is a top-level API breaking change, we might need to mark it deprecated for this major version and only remove in the next major version like Java SDK 2.x.x. What do you think?